plugin.video.on_aol/addon.xml

```diff
@@ -1,27 +1,27 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.on_aol"
        name="On Aol" 
-       version="1.1.2" 
+       version="2.0.0" 
        provider-name="divingmule">
   <requires>
     <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Please set xbmc.python to 2.14.0 for Helix .
Rest of the PR looks fine, except we're kinda discouraging the use of try-excepts as the go to instead of if-then-else blocks. Many exceptions go away using .get() in place of just the key. 

i only point it out because so many people use your code as a starting place for their own add-ons.

1.